lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -845,6 +868,10 @@ def __init__(self, client):
         self.ulimits = self._parse_ulimits()
         self.sysctls = self._parse_sysctls()
         self.log_config = self._parse_log_config()
+
+        if self.healthcheck:
+            self.healthcheck = self._parse_healthcheck_options()
```

Comment:
Remove the `if` and make `_parse_healthcheck_options()` handle `self.healthcheck == None`.